frontend/src/components/ApplicationCard.jsx

```
@@ -268,40 +268,39 @@ const ApplicationCard = ({ app, onStatusChange }) => {
+          </a>
+
+          {(status === 'PENDING' || status === 'UNDER-REVIEW') && 
+          app.postId.userId._id === user.userId && (
+            <button
+              onClick={() => setModalState({
+                statusChangeModal: true,
+                confirmationModal: false,
+                selectedStatus: null
+              })}
               className="flex items-center justify-center px-4 py-2 rounded-lg bg-gradient-to-r from-indigo-400 to-purple-500 text-white hover:opacity-90 transition-all duration-300"
             >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)